TypeCobol.LanguageServer.Test/LSRTest.cs

```diff
@@ -481,5 +488,11 @@ public void LSRAutoReplaceSecurityTest()
                 Assert.Fail("Set AutoReplace to false in UnitTestHelper.CompareDirectory()\n\n");
             }
         }
+
+        [TestMethod]
+        public void WhenCompletion()
+        {
+            LSRTestHelper.Test("WhenCompletion", LsrTestingOptions.NoLsrTesting, true);
+        }        
```

Comment:
Why 2 LSR tests ? Original test from @LANOYDO did not cover all cases ? If so maybe you should keep yours only.